.github/workflows/test.yml

```
@@ -0,0 +1,55 @@
+          #   variant: nightly
+          #   balVersion: nightly-2201.13.x
+          - name: Windows
+            os: windows-latest
+            variant: stable
+            balVersion: 2201.13.3
+          # - name: Windows
+          #   os: windows-latest
+          #   variant: nightly
+          #   balVersion: nightly-2201.13.x
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)